scripts/drone/steps/lib.star

```diff
@@ -487,6 +487,7 @@ def codespell_step():
             # Important: all words have to be in lowercase, and separated by "\n".
             'echo -e "unknwon\nreferer\nerrorstring\neror\niam\nwan" > words_to_ignore.txt',
             'codespell -I words_to_ignore.txt docs/',
+            'rm words_to_ignore.txt',
```

Comment:
must remove, otherwise it hangs over to the other pipeline and the git cleanness checker fails

Reply:
Makes sense 👍